src/dash-table/syntax-tree/SingleColumnSyntaxTree.ts

```diff
@@ -15,13 +21,22 @@ import {
 
 import columnLexicon from './lexicon/column';
 
-function getImplicitLexeme(type: ColumnType = ColumnType.Any): ILexemeResult {
+function getImplicitLexeme(
+    filterOptions: FilterCase | undefined,
+    type: ColumnType = ColumnType.Any
+): ILexemeResult {
+    const flags = R.isNil(filterOptions)
+        ? ''
+        : filterOptions === FilterCase.Insensitive
+        ? 'i'
+        : 's';
```

Comment:
Only apply flags to implicit relational operators. If the user wrote it, we must assume that's what they want!

Reply:
I'm of the opinion that's it's very readable. @alexcjohnson did agree when we last discussed it. It is somewhat controversial 😆